cli/chaosli/cmd/root.go

```diff
@@ -88,6 +95,71 @@ func initConfig() {
 	}
 }
 
+// initLibrary copies the binary-embedded disruption API into a custom folder in GOPATH
+func initLibrary() {
+	if _, isGoInstalled := os.LookupEnv("GOPATH"); !isGoInstalled {
+		log.Fatal("Setup error: please make sure go (1.11 or higher) is installed and the GOPATH is set")
+	}
+
+	err := os.Setenv("GO111MODULE", "off")
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	folderPath := fmt.Sprintf("%v/src/%v/", os.Getenv("GOPATH"), API_LIB_PATH)
+	err = os.MkdirAll(folderPath, 0750)
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	err = pkger.Walk("github.com/DataDog/chaos-controller:/api/v1beta1", func(path string, info os.FileInfo, err error) error {
```

Comment:
Maybe a small comment on what this anonymous function does would be helpful for further maintenance